### ISSUE TITLE
Tail call analysis of imported addresses

### ISSUE THREAD

**Issue by DatBrick** [Vector35/binaryninja-api#1051]
Binary Ninja Version: 1.1.1215-dev, 76179fb5
Platform: Windows 10 (10.0)

Tail call analysis does not appear to be performed for imported addresses.

![image](https://user-images.githubusercontent.com/6098371/40772204-3a1d5a70-64b7-11e8-87dd-2873ee43f731.png)

```
>>> current_mlil[3]
<il: jump(KERNEL32!LeaveCriticalSection)>
>>> current_mlil[3].operation
<MediumLevelILOperation.MLIL_JUMP: 47>
>>> current_mlil[3].get_reg_value('rsp')
<stack frame offset 0x0>
>>> current_mlil[3].dest
<il: KERNEL32!LeaveCriticalSection>
>>> current_mlil[3].dest.value
<imported address from entry 0x1417a505c>
```

**Comment by bpotchik** [comment]
Fixed in build 1.1.1216-dev.

**Comment by gaasedelen** [comment]
For general visibility & posterity, I want to note that this change regressed Ethersplay: https://github.com/trailofbits/ethersplay/pull/36

**Comment by bpotchik** [comment]
There was an unfortunate regression included with the fix in 1216.
The regression is fixed in 1.1.1218-dev.
 @gaasedelen can you report back that 1.1.1218-dev fixes Ethersplay.

**Comment by gaasedelen** [comment]
Confirming that 219 fixed the regression encountered with Ethersplay.

Thanks!
